command/pipelines/pipeline_execute.go

```diff
@@ -91,7 +113,27 @@ func (c *PipelineExecuteCommand) Run(args []string) int {
 		return 1
 	}
 
-	c.ApiMeta.Ui.Output(c.ApiMeta.Colorize().Color(fmt.Sprintf("[reset][bold][green]Pipeline execution started")))
+	executions := make([]interface{}, 0)
+	attempts := 0
+	for len(executions) != 1 && attempts < 5 {
+		executions, resp, err = c.queryExecution()
+		attempts += 1
+	}
+	if err != nil {
+		c.ApiMeta.Ui.Error(fmt.Sprintf("%s\n", err))
+		return 1
+	}
+	if resp.StatusCode != http.StatusOK {
```

Comment:
Is it possible we return 2xx but non 200 on success?

Reply:
I don't think so, but I don't think it hurts to check for 2xx here.